libs/scripts/taxonomy.js

```diff
@@ -83,7 +83,7 @@ function parseTaxonomyJson(data, root, route) {
       : (level2 ? LEVEL_INDEX.level2
         : LEVEL_INDEX.level1);
 
-    const name = level3 || level2 || level1;
+    const name = level3?.toLowerCase() || level2?.toLowerCase() || level1?.toLowerCase();
```

Comment:
Nit: seems cleaner to lowercase `name` after it's set but not a big deal

Reply:
Good catch, thank you both!